Mathlib/Algebra/Category/ModuleCat/Monoidal/Closed.lean

```diff
@@ -34,7 +34,8 @@ def monoidalClosedHomEquiv (M N P : ModuleCat.{u} R) :
   left_inv f := by
     apply TensorProduct.ext'
     intro m n
-    rw [coe_comp, Function.comp_apply, MonoidalCategory.braiding_hom_apply,
+    -- This used to be `rw`, but we need `erw` after leanprover/lean4#2644
+    erw [coe_comp, Function.comp_apply, MonoidalCategory.braiding_hom_apply,
```

Comment:
There is some strange behavior here. If I break out the terms that need `erw` then the following closes the goal alone 
```lean
    erw [coe_comp]
    rw [Function.comp_apply] 
    erw [MonoidalCategory.braiding_hom_apply, TensorProduct.lift.tmul] 
```